src/Kernel-Tests/TestDelayBasicSchedulerMicrosecondTicker.class.st

```diff
@@ -243,8 +243,8 @@ TestDelayBasicSchedulerMicrosecondTicker >> testPerformanceInlined_ifNotNil [
 { #category : #tests }
 TestDelayBasicSchedulerMicrosecondTicker >> testPerformanceInlined_ifTrue [
 	"For optimal performance, the outer conditionals of #timingPriorityHandleEvent should be inlined"
-	|codeSample|
-	codeSample := [ nil ifTrue: [ 0 ]].
+	| codeSample x |
+	codeSample := [ x ifTrue: [ 0 ]].
 	self assert: codeSample sourceNode body statements first isInlined
```

Comment:
GemStone's compiler has a special check for a literal receiver for an #ifTrue: message. This change works the same in Pharo but uses a variable rather than a literal. It still checks for the statement being inlined.

Reply:
we could even remove these methods... #timingPriorityHandleEvent does not exist anymore, the test should have been written to test the real methods. Like this it makes not much sense.

(removing them would be another PR)